lib/parameter_substitution/formatters/if_truthy.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ParameterSubstitution::Formatters::IfTruthy < ParameterSubstitution::Formatters::Base
+  def self.description
+    "The input is truthy (i.e. true, 1, y, YES) then the input is replaced with the first argument. Otherwise, the input is replaced with the second argument."
+  end
+
+  def self.has_parameters?
+    true
+  end
+
+  def initialize(value_if_true, value_if_false)
+    @value_if_true = value_if_true
+    @value_if_false = value_if_false
+  end
+
+  def format(value)
+    if [true, "true", 1, "1", "y", "yes"].include?(downcase_if_string(value))
```

Comment:
Is there a fancier way to do this? I feel like this problem has been solved a million times already in our code. Do we have it extracted somewhere?

Reply:
I'm not finding a spot where we've extracted it. I did find [this](https://github.com/Invoca/web/blob/master/lib/machine_learning/common.rb#L169-L171), which made me wonder if we want to include "t" here in this list.

Reply:
Activesupport has Boolean casters for this

Reply:
Updated